zerver/actions/realm_playgrounds.py

```diff
@@ -28,14 +28,13 @@ def do_add_realm_playground(
     acting_user: Optional[UserProfile],
     name: str,
     pygments_language: str,
-    url_prefix: str,
+    url_template: str,
 ) -> int:
     realm_playground = RealmPlayground(
         realm=realm,
         name=name,
         pygments_language=pygments_language,
-        url_prefix=url_prefix,
-        url_template=url_prefix + "{code}",
+        url_template=url_template,
     )
     # We expect full_clean to always pass since a thorough input validation
     # is performed in the view (using check_url, check_pygments_language, etc)
```

Comment:
This comment is no longer correct, since we now do the validation of URL templates only in `full_clean`, right?

Reply:
I think we might want a replacement comment at the top of this function noting that unlike most of our `do_` functions, this raises `ValidationError` 

Or maybe we should just refactor to not do that -- rename to `check_add_realm_playground` and move the `JsonableError` raising logic here.

Reply:
Right. Changed `check_url` to `url_template_validator`.

Reply:
(`check_*` being our naming convention for `do_` style functions that can fail, and might throw `JsonableError`).

Reply:
Just to clarify -- the point is that the comment is now wrong -- the view function isn't doing any validation now.

Reply:
Ah, I see. Added the refactoring change in a follow-up commit.